presto-docs/src/main/sphinx/release/release-0.243.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.243
+=============
+
+**Highlights**
+==============
+* Add function ``truncate(x, n)``.
+* Add aggregation function ``approxMostFrequent``.
+* Add session property ``check_access_control_on_utilized_columns_only``, which, when enabled, only performs access control checks on columns that would actually be required to produce the query output, ignoring columns that are referenced in the query, but are not required to compute the query results.
+* Add session property ``remote_functions_enabled`` to configure whether remote functions are allowed.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add function ``truncate(x, n)``.
+* Add aggregation function ``approxMostFrequent``.
+* Fix query failure when explaining ``Use`` statement.
+* Add session property ``check_access_control_on_utilized_columns_only``, which, when enabled, only performs access control checks on columns that would actually be required to produce the query output, ignoring columns that are referenced in the query, but are not required to compute the query results.
```

Comment:
Following the guideline in https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#phrasing - this should be phrased as (also, we can shorten it)

```
Add support for ignoring access checks on columns that are referenced in the query,
but are not required to compute the query results. This can be enabled by the session .....
```

@prithvip for his thoughts on this framing.

Reply:
Also, we should move this to `Security changes` section - https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-sections

Reply:
Consider rewrite:
> Add session property check_access_control_on_utilized_columns_only, which toggles performing access control checks on columns that are used to produce the query output thereby ignoring columns that are referenced in the query, but are not required to compute the query results.